### PR TITLE
JPN-577 | updated Help module

### PR DESCRIPTION
**Pull request by d34th4ck3r**
@Wikia/spitfires @perjg

### CODE REVIEW

File: extensions/wikia/CommunityPage/models/CommunityPageSpecialHelpModel.class.php [hunk 1]
```diff
@@ -4,14 +4,30 @@ class CommunityPageSpecialHelpModel {
...
+			'editPage' => wfMessage( 'communitypage-help-edit-page' )->plain(),
+			'addLinks' => wfMessage( 'communitypage-help-add-link' )->plain(),
+			'addNewPage' => wfMessage( 'communitypage-help-add-new-page' )->plain(),
+			'editPageLink' => $this->geteditPageLink(),
+			'addLinksPageLink' => $this->getAddLinksPageLink(),
+			'addNewPageLink' => $this->getAddNewPageLink()
 		];
 	}
+	
+	private function geteditPageLink() {
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialHelpModel.class.php]: What do you think about creating one method and pass message key, for example `getHelpPageLink(messageKey`)?

File: extensions/wikia/CommunityPage/templates/helpModule.mustache [hunk 2]
```diff
@@ -1,4 +1,6 @@
 <div class="module help-module">
 	<h2>{{helpModule.title}}</h2>
-	<a data-tracking="help-page-link" href="{{helpModule.link}}">{{helpModule.linkText}}&nbsp;&rarr;</a>
+    <a data-tracking="help-page-link" href="{{helpModule.editPageLink}}">{{helpModule.editPage}}&nbsp;&rarr;</a><br/>
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/templates/helpModule.mustache]: whitespaces instead of tabs

**Comment by d34th4ck3r** [comment]
I have made changes according to your comments!

**Comment by lukaszkonieczny** [comment]
LGTM

**Pull request merged**